packages/grafana-data/src/types/explore.ts

```diff
@@ -6,12 +8,11 @@ type AnyQuery = DataQuery & Record<string, any>;
 
 /** @internal */
 export interface ExploreUrlState<T extends DataQuery = AnyQuery> {
-  datasource: string;
+  datasource: string | DataSourceRef;
```

Comment:
Where do we use `DataSourceRef` instead of a string? Can't seem to find it 🤔

Reply:
nowhere, must be a leftover, good catch! fixed in 6be28b8e9d